tests/test_models.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Test the creation of dataclasses and DTOs"""
+
+from datetime import datetime, timezone
+
+from wps.core.models import WorkOrderToken, WorkPackage, WorkType
+
+
+def test_work_order_token():
```

Comment:
If we don't implement custom validators, I think we don't need to test model initialization and just rely on the pydantic that it is working correctly.

Reply:
Yes, it's a bit redundant. I think it can still help to check that all types have been set properly. But then negative tests would be even more helpful, e.g. to make sure that only valid enum values can been passed, i.e. these fields have not been accidentally declared as str. This happened to me recently, it went unnoticed and caused some confusion later.